src/ontology/uberon-edit.obo

```diff
@@ -225615,6 +225615,20 @@ relationship: in_taxon NCBITaxon:8782 ! Aves
 property_value: http://purl.org/dc/elements/1.1/date 2022-01-24T16:47:47Z xsd:dateTime
 created_by: http://orcid.org/0000-0002-2825-0621
 
+[Term]
+id: UBERON:8410070
+name: levatores costarum
+def: "A chest muscle with a role in forceful inspiration. In human, there are twelve levatores costarum on either side of the chest; they are small tendinous and fleshy bundles arising from the ends of the transverse processes of the seventh cervical and upper eleven thoracic vertebrae. They pass obliquely downward and laterally, like the fibers of the intercostales externi, and each is inserted into the outer surface of the rib immediately below the vertebra from which it takes origin, between the tubercle and the angle (levatores costarum breves). Each of the four lower muscles divides into two fasciculi, one of which is inserted as above described; the other passes down to the second rib below its origin (levatores costarum longi)." [WikipediaVersioned:Levatores_costarum_muscles&oldid=876138804]
+synonym: "levatores costarum muscle" EXACT []
+synonym: "musculi levatores costarum" EXACT LATIN []
+xref: FMA:71312
+is_a: UBERON:0002426 ! chest muscle
+relationship: has_muscle_insertion UBERON:0002228 ! rib
+relationship: has_muscle_origin UBERON:0001077 ! transverse process of vertebra
+property_value: http://purl.org/dc/elements/1.1/date 2022-01-27T10:41:44Z xsd:dateTime
+property_value: present_in_taxon "Homo sapiens" xsd:string
```

Comment:
Should use NCBI_taxon IRI of Homo Sapiens as value.

Reply:
Just saw this... will fix, but what about all the others?